plugins/inventory/proxmox.py

```diff
@@ -70,6 +70,13 @@
         description: Gather LXC/QEMU configuration facts.
         default: no
         type: bool
+      want_proxmox_nodes_ansible_host:
+        version_added: 2.5.1
```

Comment:
We do not add new features in bugfix releases. Also 2.5.1 has already been released, you cannot add features **after** a release.
```suggestion
        version_added: 3.0.0
```

Reply:
Usually you can take the version in `galaxy.yml` in `main` branch: https://github.com/ansible-collections/community.general/blob/main/galaxy.yml#L3

Reply:
didn't know, will know for next time. tnx.